config/stack.go

```diff
@@ -11,14 +11,18 @@ func Stacks() []string {
 		"osx-vs4mac-beta",
 		"osx-vs4mac-previous-stable",
 		"osx-vs4mac-stable",
-		"osx-xcode-10.0.x",
 		"osx-xcode-10.1.x",
 		"osx-xcode-10.2.x",
```

Comment:
These stacks are no longer valid.

Reply:
The problem is that currently, the test is checking the available stacks at https://github.com/bitrise-io/bitrise.io/tree/master/system_reports, which contains these outdated versions. Since we will change how BANP is retrieving the default stack version, I would change how it handles the available stacks as well. Until then we either:

- remove the unsupported stacks and disable this test,
- or, keep the unsupported stacks and keep the test.

Which one would you prefer?

Reply:
We can keep them as long they are not selectable easily from the list.